app/src/main/java/com/geeksville/mesh/ui/UsersFragment.kt

```diff
@@ -205,8 +205,8 @@ class UsersFragment : ScreenFragment("Users"), Logging {
 
             holder.lastTime.text = formatAgo(n.lastHeard)
 
-            if (n.envMetricStr.isNotEmpty()) {
-                holder.envMetrics.text = n.envMetricStr
+            if (n.envMetricStr(model.module.telemetry.environmentDisplayFahrenheit).isNotEmpty()) {
+                holder.envMetrics.text = n.envMetricStr(model.module.telemetry.environmentDisplayFahrenheit)
```

Comment:
make sure to check your added code follows [coding conventions](https://kotlinlang.org/docs/coding-conventions.html). to reformat the selected text In Android Studio there is the Ctrl + Alt + L hotkey.

you could also store `n.envMetricStr(model.module.telemetry.environmentDisplayFahrenheit` in a variable before the if/else block so it is only called once.